apps/web-giddh/src/assets/css/style-2.scss

```diff
@@ -11175,6 +11175,15 @@ inventory-master {
     }
 }
 
+/* template froala */
+body {
+    &.hide-chat-widget {
+        chat-widget {
+            display: none !important;
```

Comment:
@dvCodeWorld according to me this is not proper solution.
Please confirm for this

Reply:
confirm